mmv1/third_party/terraform/services/netapp/resource_netapp_kmsconfig_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package netapp_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccNetappkmsconfig_kmsConfigCreateExample_Update(t *testing.T) {
+	t.Parallel()
```

Comment:
```suggestion
```

Reply:
Woot! This is a missing link I was looking for in the last weeks. We have multiple resources (Active Directory, KSMconfig, BackupVault) which are unique per region and tests are failing when ran in parallel. We worked around that so far, but this is the best way to solve that. I learned something important. Thank you!

Reply:
But I am still surprised the tests did fail. We ran them in two different regions, so they shouldn't impact each other. Let us incorporate your suggestions and see how it works. My local test works fine.